MonoGame.Framework/Content/ContentManager.cs

```diff
@@ -372,18 +374,83 @@ public virtual T Load<T>(string assetName)
 
             // Load the asset.
             result = ReadAsset<T>(assetName, null);
-
             loadedAssets[key] = result;
+
             return result;
 		}
 
+        /// <summary />
+		protected T ReadAsset<T>(string assetName, Action<IDisposable> recordDisposableObject)
```

Comment:
In the future @titanix avoid re-ordering functions like this.  It makes it hard to track the actual code changes when the diff no longer lines up in history.

Reply:
@titanix could you please move it back.